src/app/productConfig/productConfigModal/productConfig.modal.component.js

```diff
@@ -183,10 +178,11 @@ class ModalInstanceCtrl {
     savingObservable.subscribe( () => {
       if ( this.isEdit ) {
         this.$scope.$emit( cartUpdatedEvent );
-        this.$uibModalInstance.close( {isUpdated: true} );
+        this.close( {isUpdated: true} );
+        this.close( {isUpdated: false} );
```

Comment:
That probably came from a merge conflict. I had started this awhile ago. I'll fix it. Good catch.